src/languageHandler.tsx

```diff
@@ -90,6 +90,22 @@ const translateWithFallback = (text: string, options?: object): { translated?: s
     const translated = counterpart.translate(text, { ...options, fallbackLocale: counterpart.getLocale() });
     if (!translated || /^missing translation:/.test(translated)) {
         const fallbackTranslated = counterpart.translate(text, { ...options, locale: FALLBACK_LOCALE });
+        if (!fallbackTranslated || /^missing translation:/.test(translated) && process.env.NODE_ENV !== "development") {
```

Comment:
Why a regex over `translated.startsWith(...)`?

Reply:
consistency with the existing code (I guess that might predate startsWith)

Reply:
I've replaced with `startsWith` which I think is fine, and along the way realised the second test was erroneously against `translated`, not `fallbackTranslated` ..